examples/helia-101/README.md

```diff
@@ -194,40 +223,109 @@ const libp2p = await createLibp2p({
 })
 ```
 
-### 401 - Providing
+### 302 - Local Peer Discovery
+
+The [302-mdns.js](./302-mdns.js) example demonstrates how to use multicast DNS for local peer discovery.
+
+This allows Helia nodes on the same local network to automatically discover each other without requiring bootstrap nodes or DHT lookups.
+
+```js
+import { mdns } from '@libp2p/mdns'
+
+// Configure libp2p with mDNS discovery
+const libp2p = await createLibp2p({
+  // ... other configuration ...
+  peerDiscovery: [
+    mdns()
+  ],
+  // ... rest of configuration ...
+})
+
+// Listen for peer discovery events
+node.libp2p.addEventListener('peer:discovery', (evt) => {
+  console.log(`Discovered new peer (${evt.detail.id.toString()}) via MDNS`)
+  node.libp2p.dial(evt.detail.multiaddrs)
+})
+```
+
+To run this example, use the following command:
+
+```console
+> npm run 302-mdns
+```
+
+### 303 - Prometheus Metrics
+
+The [303-metrics.js](./303-metrics.js) example shows how to enable and expose Prometheus metrics for your Helia node, and how to expose the Prometheus HTTP metrics endpoint.
```

Comment:
Maybe worth mentioning other metrics impls that are available?  DevTools metrics for browsers and Otel metrics (which includes a prometheus/graphana compatible scraping endpoint).